main.qml

```diff
@@ -529,23 +531,37 @@ ApplicationWindow {
         walletName = usefulName(wallet.path)
         console.log(">>> wallet opened: " + wallet)
         if (wallet.status !== Wallet.Status_Ok) {
-            // try to resolve common wallet cache errors automatically
-            switch (wallet.errorString) {
-                case "basic_string::_M_replace_aux":
-                case "std::bad_alloc":
-                    walletManager.clearWalletCache(wallet.path);
-                    walletPassword = passwordDialog.password;
-                    appWindow.initialize();
-                    console.error("Repairing wallet cache with error: ", wallet.errorString);
-                    appWindow.showStatusMessage(qsTr("Repairing incompatible wallet cache. Resyncing wallet."),6);
-                    return;
-                default:
-                    // opening with password but password doesn't match
-                    console.error("Error opening wallet with password: ", wallet.errorString);
-                    passwordDialog.showError(qsTr("Couldn't open wallet: ") + wallet.errorString);
-                    console.log("closing wallet async : " + wallet.address)
-                    closeWallet();
-                    return;
+            if(wallet.errorString == "Could not connect to the device Trezor" || wallet.errorString == "Wrong Device Status: 0x6e00 (SW_CLA_NOT_SUPPORTED), EXPECTED 0x9000 (SW_OK), MASK 0xffff") {
```

Comment:
0x6e00 is not the only possible Ledger error status.

Reply:
Can you inform me what are the error codes when 1) ledger is locked and 2) ledger app is not open?